common/djangoapps/xblock_django/tests/test_models.py

```diff
@@ -0,0 +1,58 @@
+"""
+Tests for deprecated xblocks in XBlockDisableConfig.
+"""
+import ddt
+
+from mock import patch
+from django.test import TestCase
+from xblock_django.models import XBlockDisableConfig
+
+
+@ddt.ddt
+class XBlockDisableConfigTestCase(TestCase):
+    """
+    Tests for the DjangoXBlockUserService.
+    """
+    def setUp(self):
+        super(XBlockDisableConfigTestCase, self).setUp()
+
+        # Initialize the deprecated modules settings with empty list
+        XBlockDisableConfig.objects.create(
+            disabled_blocks='', enabled=True
+        )
```

Comment:
@ehteshamkafeel Why we need to create this? As per my understanding its not needed.